src/logs/logs_sender.py

```diff
@@ -147,7 +148,8 @@ def ensure_content_length(log_entry, context):
     log_content_len = len(log_content)
     if log_content_len > DYNATRACE_LOG_INGEST_CONTENT_MAX_LENGTH:
         context.sfm.log_content_trimmed()
-        log_entry["content"] = log_content[0: DYNATRACE_LOG_INGEST_CONTENT_MAX_LENGTH]
+        trimmed_len = DYNATRACE_LOG_INGEST_CONTENT_MAX_LENGTH - len(DYNATRACE_LOG_INGEST_CONTENT_MARK_TRIMMED)
```

Comment:
I don't know if Python would somehow optimize it under the hood (to not calculate `trimmed_len` every time). I suggest simply creating a `DYNATRACE_LOG_INGEST_CONTENT_TRIMMED_LENGTH` constant.

Reply:
I'll take a look.

Reply:
This would only be calculated for the very long logs (so I assume quite rarely). Moreover, the `len()` function takes constant time, so it's basically just two operations. I think compute time optimization is not really important here. We can add a constant though, just to make it clearer to the person reading the code, that it always computes to the same value

Reply:
As we discussed, I'll leave current implementation. The time complexity of len(string) is constant O(1), and it's a simple math operation.